src/PhpSpec/Message/MessageInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Message;
+
+interface MessageInterface
```

Comment:
I don't think this interface is really needed. 'Message' is a very generic concept; you would be better off naming the methods in `CurrentExample` to `setCurrentExample` and `getCurrentExample`.